dm/dm/ctl/master/source_table_schema.go

```diff
@@ -23,7 +25,7 @@ import (
 // NewSourceTableSchemaCmd creates a SourceTableSchema command.
 func NewSourceTableSchemaCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "binlog-schema <task-name> <table-filter1> <table-filter2> ...",
+		Use:   "binlog-schema <task-name> <database> <table>",
```

Comment:
Also recommend add a `binlog-schema list` subcommand 🤔

Reply:
ok, add `list` like `binlog` command will be better.